homeassistant/components/comfoconnect/climate.py

```diff
@@ -0,0 +1,136 @@
+"""Platform to control a Zehnder ComfoAir Q350/450/600 ventilation unit."""
+import logging
+
+from pycomfoconnect import (
+    CMD_BYPASS_OFF,
+    CMD_BYPASS_ON,
+    SENSOR_BYPASS_STATE,
+    SENSOR_TEMPERATURE_SUPPLY,
+)
+
+from homeassistant.components.climate import (
+    HVAC_MODE_COOL,
+    HVAC_MODE_HEAT,
+    TEMP_CELSIUS,
+    ClimateEntity,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from . import DOMAIN, SIGNAL_COMFOCONNECT_UPDATE_RECEIVED, ComfoConnectBridge
+
+_LOGGER = logging.getLogger(__name__)
+HVAC_MODES = [HVAC_MODE_HEAT, HVAC_MODE_COOL]
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the ComfoConnect climate platform."""
+    if discovery_info is None:
+        return
+
+    ccb = hass.data[DOMAIN]
+    add_entities([ComfoConnectBypass(ccb.name, ccb)], True)
+
+
+class ComfoConnectBypass(ClimateEntity):
+    """Representation of the ComfoConnect bypass platform."""
+
+    def __init__(self, name, ccb: ComfoConnectBridge) -> None:
+        """Initialize the ComfoConnect bypass."""
+        self._name = name
```

Comment:
I assume this is meant to be used as the Entity's friendly name?
```suggestion
        self._attr_name = name
```